erpnext/manufacturing/doctype/production_plan/production_plan.py

```diff
@@ -639,6 +639,9 @@ def get_sub_assembly_items(self, manufacturing_type=None):
 		sub_assembly_items_store = []  # temporary store to process all subassembly items
 
 		for row in self.po_items:
+			if not row.item_code:
+				frappe.throw(_(f"Row #{row.idx}: Please select Item Code in Assembly Items"))
```

Comment:
```suggestion
				frappe.throw(_("Row #{0}: Please select Item Code in Assembly Items").format(row.idx))
```
F-strings don't work with translations.